src/hb-buffer.cc

```diff
@@ -448,27 +448,13 @@ void
 hb_buffer_t::reverse_range (unsigned int start,
 			    unsigned int end)
 {
-  unsigned int i, j;
-
   if (end - start < 2)
     return;
 
-  for (i = start, j = end - 1; i < j; i++, j--) {
-    hb_glyph_info_t t;
-
-    t = info[i];
-    info[i] = info[j];
-    info[j] = t;
-  }
+  hb_array_t<hb_glyph_info_t> (info, len).reverse (start, end);
```

Comment:
Again.  I meant `hb_array (info, len).reverse (start, end);`.  `hb_array` is a function that deduces the template type for you so you don't have to type it.